booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java

```diff
@@ -659,10 +659,18 @@ private static void generateScenarioRelatedChecks(List<JsonContentCheck> checks,
     JsonAttribute.mustBeDatasetKeywordIfPresent(JsonPointer.compile("/incoTerms"), BookingDataSets.INCO_TERMS_VALUES),
     JsonAttribute.allIndividualMatchesMustBeValid(
       "The 'cargoGrossVolume' implies 'cargoGrossVolumeUnit'",
-      mav -> mav.submitAllMatching("requestedEquipments.*.commodities.*"),
+      mav -> mav.submitAllMatching("requestedEquipments.*.commodities.*.cargoGrossVolume"),
       JsonAttribute.presenceImpliesOtherField(
-        "cargoGrossVolume",
-        "cargoGrossVolumeUnit"
+        "value",
+        "unit"
+      )
+    ),
+    JsonAttribute.allIndividualMatchesMustBeValid(
+      "The 'cargoGrossWeight' implies 'cargoGrossWeightUnit'",
+      mav -> mav.submitAllMatching("requestedEquipments.*.commodities.*.cargoGrossWeight"),
+      JsonAttribute.presenceImpliesOtherField(
+        "value",
+        "unit"
       )
     ),
```

Comment:
I do not think this is necessary? I thought `value` and `unit` were both mandatory in this kind of object.

Reply:
Nvm, fixed in a later commit